cfpb/consumerfinance.gov
# Don't highlight first FIG menu item on page load

This fixes an a11y bug where the page's focus was being stolen on page load. Plus, smaller screens don't even have the first section visible in the viewport on page load so it doesn't make sense to have it highlighted by default.

## Changes

- FIG scrollspy page load behavior + relevant tests
- Removed unnecessary duration from unrelated spec
- Unrelated: Removed unnecessary `async` attribute from `<script>` element.

## How to test this PR

(This branch has been deployed to dev4.)

1. Pull down this branch.
1. Functional tests should pass:
    `yarn cypress run --spec test/cypress/integration/compliance/small-business-lending/filing-instruction-guide.cy.js --env FIG_URL=/data-research/small-business-lending/filing-instructions-guide/filing-instructions-guide-test/ --config baseUrl=https://<secret-dev4-subdomain>.cfpb.gov`

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
